### PR TITLE
Remove libdohj dependency

### PR DESCRIPTION
**Pull request by oscarguindzberg**
Remove libdohj dependency and the code to use LTC, DASH or DOGE as base currencies.

### CODE REVIEW

File: core/src/main/java/bisq/core/app/WalletAppSetup.java [hunk 1]
```diff
@@ -90,8 +88,8 @@ void init(@Nullable Consumer<String> chainFileLockedExceptionHandler,
               Runnable walletPasswordHandler,
               Runnable downloadCompleteHandler,
               Runnable walletInitializedHandler) {
-        log.info("Initialize WalletAppSetup with BitcoinJ version {} and LibDohJ version {} with hash of BitcoinJ commit {}",
-                VersionMessage.BITCOINJ_VERSION, Version.VERSION, Version.BITCOINJ_VERSION);
+        log.info("Initialize WalletAppSetup with BitcoinJ version and hash of BitcoinJ commit {}",
```
Comment (ManfredKarrer) [inline core/src/main/java/bisq/core/app/WalletAppSetup.java]: I think here is missing the {} -> BitcoinJ version {} and

**Review by ManfredKarrer — CHANGES_REQUESTED** [review]

File: core/src/main/java/bisq/core/app/WalletAppSetup.java (hunk 1 above)
Comment (ManfredKarrer) [reply core/src/main/java/bisq/core/app/WalletAppSetup.java]: Beside that, ACK (just tested).

**Review by ripcurlx — CHANGES_REQUESTED** [review]
ACK - besides the small change requested by @ManfredKarrer

File: core/src/main/java/bisq/core/app/WalletAppSetup.java (hunk 1 above)
Comment (oscarguindzberg) [reply core/src/main/java/bisq/core/app/WalletAppSetup.java]: Done. just force pushed the fix.

**Review by ManfredKarrer — APPROVED** [review]
utACK

**Pull request merged**